3-0-libraries.md

```diff
@@ -248,14 +248,73 @@ summed when displayed on this page.
 
 # Troubleshooting
 
-1. How do I correct an index on a library?
-1. What if I assign a library to the wrong parent aliquot?
-1. What if I forget to put a library dilution in a pool?
-1. How do I change the targeted sequencing type on a library?
-1. How can I add a new targeted sequencing type, kit, or anything else in drop-down menus?
-1. How do I make bulk orders?
-1. How are matrix tube barcodes assigned to tubes?
-1. What is the importance of selecting a study (?) since it prevents the Sequencing stage from adding pools to the lane containers?
+## How do I correct an index on a library?
+The index can be changed on either the individual library page or the bulk edit page.
+
+1. From _Tracking_, _Library_, select the library that needs to be changed.
+1. If necessary, change _Index Family_. If selected, the indices will be erase.
+1. Select the correct index under _Indicies_. If the family supports it, another drop down will appear.
+
+## What if I assign a library to the wrong parent aliquot?
+Please email gsi@oicr.on.ca or file a JIRA ticket in GSI Common to get assistance from the MISO team.
+
+## What if I forget to put a library dilution in a pool?
+If the library dilution has not been created:
+
+1. From _Tracking_, _Library_, select the library that needs to be added.
+1. In the _Library Dilution_ section, hover over _Options_ and select _Add Library Dilution_.
+1. Fill out the new row in the table, then click _Add_.
+
+Once the dilution exists:
+
+1. From _Tracking_, _Pool_, select the pool that needs the additional library dilution.
+1. In the _Pool Elements_ section, find the second table, labelled _Select poolable elements_.
+1. Enter the library name, library alias, or dilution name in the _Search_ box.
+1. Find the correct dilution in the list and click the plus button beside it.
+
+## How do I change the targeted sequencing type on a library?
+Targeted sequencing is connected to the dilution since the same library can be used for multiple targeted sequencing panels.
+
+1. From _Tracking_, _Library_, select the library that needs to be changed.
+1. In the _Library Dilution_ section, under the _Edit_ column, click the pencil beside the dilution that needs to be changed.
+1. Select a new _Targeted Sequencing_ from the list.
+1. Under the _Edit_ column, click _Save_.
+
+## How can I add a new targeted sequencing type, kit, or anything else in drop-down menus?
+For targeted sequencing and indices, please email gsi@oicr.on.ca or file a JIRA ticket in GSI Common to get assistance from the MISO team.
+
+Kits can be added easily:
+
+1. From _Tracking_, _Kits_, select the type of kit you want to add.
+1. Click _Create Kit Descriptor_.
+1. Fill in the form. The _Stock Level_ is not currently used, so leave it at zero.
+1. Carefully select the _Kit Type_ and _Platform_. This will decide where the kit is displayed.
```

Comment:
"This will decide where the kit is displayed" -> "This will determine what the kit can be used for."